### ISSUE TITLE
Error installing virtualbox guest modules arch

### ISSUE THREAD

**Issue by ademar111190** [AnarchyLinux/installer#986]
## Basic info:

**Anarchy version**: 1.0.10-1

**Installation method (VM/Host)**: Virtual Box Mac OSX Host

**System specifications (Processor, GPU, RAM ...)**: MacBook Pro (15-inch, 2019) 2,6 GHz 6-Core Intel Core i7 16 GB 2400 MHz DDR4 Radeon Pro 555X 4 GB Intel UHD Graphics 630 1536 MB

## What you expected would happen

After setting up everything, starting the install process I expected to download and install the software

## What actually happened

The installation fails with the following error:

```
==> Creating install root at /mnt
==> Installing packages to /mnt
:: Synchronizing packages databases...
downloading core.db...
downloading extra.db...
downloading community.db...
:: There are 24 members in group base-devel:
:: Repository core
   1) autoconf  2) automake  3) binutils  4) bison  5) fakeroot  6) file  7) findutils  8) flex  9) gawk  10) gcc  11) gettext  12) grep  13) groff  14) gzip  15) libtool  16) m4  18) pacman  19) patch  20) pkgconf  21) sed  22) sudo  23) textinfo  24) which

Enter a selection (default=all): error: target not found: virtualbox-guest-modules-arch

==> Error: Failed to install packages to new root
Install failed: please report to developer
```

## Steps needed to reproduce the bug

1. Boot the image through virtual box
2. Select the XFCE
3. No swap
4. Start the installation

## Bug report checklist

* [X] I've read the [Reporting Issues Wiki Page](https://github.com/AnarchyLinux/installer/wiki/Reporting-issues)
* [X] I've removed unnecessary sections of the bug submission template
* [X] I understand that if I don't do everything listed in the checkmarks above my issue won't be considered a priority and may get closed at any time without warning, because I didn't take the time to properly submit my bug report

**Comment by erazemkokot** [comment]
Please update the installer before installing (`anarchy -u` in the main menu). This issue has already been fixed in [d2e54180df0d15aa6edd76a50c0277672623321a](https://github.com/AnarchyLinux/installer/commit/d2e54180df0d15aa6edd76a50c0277672623321a).

**Comment by ademar111190** [comment]
Thank you for the fix, it works.
